.gitmodules

```diff
@@ -32,6 +32,9 @@
 	path = cime
 	url = git@github.com:ESMCI/cime.git
 	branch = master
+[submodule "externals/ekat"]
+        path = externals/ekat
+        url = git@github.com:E3SM-Project/EKAT.git
 [submodule "externals/YAKL"]
```

Comment:
The github diff is showing a change to YAKL.  Different hash in the submodule?

Reply:
Yes the version from the scream repo appears to be slightly newer, so I think this is the version we want.